config-linux.md

```diff
@@ -371,8 +371,8 @@ The following parameters can be specified to set up the controller:
 ```json
 "cpu": {
     "shares": 1024,
-    "quota": 1000000,
-    "period": 500000,
+    "quota": 500000,
```

Comment:
Why quota needs to be smaller than period? The config seems valid on a multi-CPU machine.

Reply:
Indeed -- this change would effectively mean a single container could never use more than one core's worth of CPU time.

When I use Docker's `--cpus` shorthand for setting these options, here's what `/sys/fs/cgroup/cpu.max` looks like:

```console
$ docker run --rm --cpus 2 bash cat /sys/fs/cgroup/cpu.max
200000 100000
$ docker run --rm --cpus 0.5 bash cat /sys/fs/cgroup/cpu.max
50000 100000
```

(Both very legal and sane configurations.)

Reply:
Never mind, not saying it was a mistake, just figured quota less than period would be more understandable, extra description might needed to explain the multi cpu case (like examples in: https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt) which I thought can be avoided.

Thanks for your comments.